django/contrib/admin/static/admin/js/actions.js

```diff
@@ -51,29 +51,33 @@
         }
         actionCheckboxes.forEach(function(el) {
             el.checked = checked;
-            el.closest('tr').classList.toggle(options.selectedClass, checked);
+            if (!options.inline) {
+                el.closest('tr').classList.toggle(options.selectedClass, checked);
+            }
         });
     }
 
     function updateCounter(actionCheckboxes, options) {
-        const sel = Array.from(actionCheckboxes).filter(function(el) {
-            return el.checked;
-        }).length;
-        const counter = document.querySelector(options.counterContainer);
-        // data-actions-icnt is defined in the generated HTML
-        // and contains the total amount of objects in the queryset
-        const actions_icnt = Number(counter.dataset.actionsIcnt);
-        counter.textContent = interpolate(
-            ngettext('%(sel)s of %(cnt)s selected', '%(sel)s of %(cnt)s selected', sel), {
-                sel: sel,
-                cnt: actions_icnt
-            }, true);
-        const allToggle = document.getElementById(options.allToggleId);
-        allToggle.checked = sel === actionCheckboxes.length;
-        if (allToggle.checked) {
-            showQuestion(options);
-        } else {
-            clearAcross(options);
+        if (!options.inline) {
+            const sel = Array.from(actionCheckboxes).filter(function(el) {
+                return el.checked;
+            }).length;
+            const counter = document.querySelector(options.counterContainer);
+            // data-actions-icnt is defined in the generated HTML
+            // and contains the total amount of objects in the queryset
+            const actions_icnt = Number(counter.dataset.actionsIcnt);
+            counter.textContent = interpolate(
+                ngettext('%(sel)s of %(cnt)s selected', '%(sel)s of %(cnt)s selected', sel), {
+                    sel: sel,
+                    cnt: actions_icnt
+                }, true);
+            const allToggle = document.getElementById(options.allToggleId);
+            allToggle.checked = sel === actionCheckboxes.length;
+            if (allToggle.checked) {
+                showQuestion(options);
+            } else {
+                clearAcross(options);
+            }
```

Comment:
```suggestion
        const sel = Array.from(actionCheckboxes).filter(function(el) {
            return el.checked;
        }).length;
        const counter = document.querySelector(options.counterContainer);
        // data-actions-icnt is defined in the generated HTML
        // and contains the total amount of objects in the queryset
        const actions_icnt = Number(counter.dataset.actionsIcnt);
        counter.textContent = interpolate(
            ngettext('%(sel)s of %(cnt)s selected', '%(sel)s of %(cnt)s selected', sel), {
                sel: sel,
                cnt: actions_icnt
            }, true);
        const allToggle = document.getElementById(options.allToggleId);
        allToggle.checked = sel === actionCheckboxes.length;
        if (allToggle.checked) {
            showQuestion(options);
        } else {
            clearAcross(options);
```
As `updateCounter` is not called for inlines, we should be able to remove the if